packages/shared/src/components/tooltips/ContextMenuItem.tsx

```diff
@@ -17,14 +18,17 @@ function ContextMenuItem({
   onClick,
   Icon,
   className,
+  target,
   ...props
 }: ContextMenuItemProps): ReactElement {
   return (
     <ConditionalWrapper
       condition={!!href}
       wrapper={(children) => (
         <Link href={href}>
```

Comment:
Not sure don't you need target here as well for nextjs?

Reply:
That should work on newer versions of Next.js. This one needs `target` on `a` to work. Tested it.